Missing compile dependency?

When trying to build the commit 042ca386f20d1fbe13e8bf42b5d39c99aa2e6a78, I get this error:
```
> Task :plugin-bazel-event-service:compileJava
/home/devel/teamcity-bazel-plugin/plugin-bazel-event-service/build/generated/source/proto/main/grpc/com/google/bytestream/ByteStreamGrpc.java:37: error: cannot find symbol
@javax.annotation.Generated(
                 ^
  symbol:   class Generated
  location: package javax.annotation
/home/devel/teamcity-bazel-plugin/plugin-bazel-event-service/build/generated/source/proto/main/grpc/com/google/devtools/build/v1/PublishBuildEventGrpc.java:35: error: cannot find symbol
@javax.annotation.Generated(
                 ^
  symbol:   class Generated
  location: package javax.annotation
Note: /home/devel/teamcity-bazel-plugin/plugin-bazel-event-service/build/generated/source/proto/main/java/com/google/devtools/build/lib/buildeventstream/BuildEventStreamProtos.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
2 errors

> Task :plugin-bazel-event-service:compileJava FAILED

FAILURE: Build failed with an exception.
```

When I add `"javax.annotation:javax.annotation-api:1.3.2"` as a compile dependency to the `plugin-bazel-event-service` subproject, the compilation error goes away.

--- a/plugin-bazel-event-service/build.gradle
+++ b/plugin-bazel-event-service/build.gradle
@@ -52,6 +52,7 @@ dependencies {
     compile "io.grpc:grpc-netty:${grpcVersion}"
     compile "io.grpc:grpc-protobuf:${grpcVersion}"
     compile "io.grpc:grpc-stub:${grpcVersion}"
+    compile "javax.annotation:javax.annotation-api:1.3.2"
     compile "org.jetbrains.teamcity:serviceMessages:${teamcityVersion}"
     compile "org.apache.logging.log4j:log4j-1.2-api:2.17.2"
     compile "org.apache.logging.log4j:log4j-core:2.17.2"

